src/main/java/com/adobe/aem/compgenerator/utils/HTMLUtils.java

```diff
@@ -0,0 +1,103 @@
+        List<Property> props = getAllProperties(generationConfig);
+        String slingModel = StringUtils.uncapitalize(generationConfig.getJavaFormatedName()) + "Model";
+        StringBuilder renderedHtml = new StringBuilder();
+
+        for (Property prop : props) {
+            String label = "label_not_defined";
+            if (prop.getLabel() != null) {
+                label = prop.getLabel();
+            }
+
+            if (Constants.TYPE_CHECKBOX.equals(prop.getType())) {
+                renderedHtml.append(generateParagraphHtml(label,
+                        prop.getField(),
+                        slingModel,
+                        " ? 'checked' : 'unchecked'"
+                ));
+            } else if (Constants.TYPE_DATEPICKER.equals(prop.getType())) {
+                renderedHtml.append(generateParagraphHtml(label,
+                        prop.getField(),
+                        slingModel,
+                        ".time.toGMTString"
+                ));
+            } else if (Constants.TYPE_IMAGE.equals(prop.getType())) {
+                renderedHtml.append(generateImageHtml(label, prop.getField(), slingModel));
+            } else if (Constants.TYPE_MULTIFIELD.equals(prop.getType())) {
+                renderedHtml.append(generateListHtml(prop, slingModel));
+            } else {
+                renderedHtml.append(generateParagraphHtml(label,
+                        prop.getField(),
+                        slingModel,
+                        StringUtils.EMPTY));
+            }
+        }
+        return renderedHtml.toString();
+    }
+
+    private static String generateImageHtml(String label, String field, String slingModel) {
+        return "<p>" +
+                label +
+                ": <img src=\"" +
+                slingModel +
+                "." +
+                field +
+                ".src}\"/></p>\n";
+    }
+
+    private static String generateListHtml(Property prop, String slingModel) {
+        String initialListHtml = "<div data-sly-list=\"${" +
+                slingModel +
+                "." +
+                prop.getField() +
+                "}\">\n\t<p>";
+        if (prop.getItems().size() > 1) {
+            StringBuilder items = new StringBuilder(initialListHtml);
+            for (Property property : prop.getItems()) {
+                items.append(property.getLabel())
+                        .append(": ${item.")
+                        .append(property.getField())
+                        .append("}\n\t");
+            }
+            return items + "</p>\n</div>\n";
+        } else {
+            return initialListHtml +
+                    prop.getLabel() +
+                    ": ${item}</p>\n</div>\n";
+        }
+    }
+
+    private static String generateParagraphHtml(String label, String field, String slingModel, String additional) {
+        return "<p>" +
+                label +
+                ": $̊̄̄{" +
```

Comment:
looks like an odd character here...like its some unicode version of `$` instead of the normal.  Or is the github UI diff just showing something odd here?

Reply:
True, it was a mistake on my end.
Fixed.